lms/services/vitalsource/service.py

```diff
@@ -7,9 +7,16 @@
 class VitalSourceService:
     """A high-level interface for dealing with VitalSource."""
 
-    def __init__(self, client: VitalSourceClient):
+    def __init__(self, client: VitalSourceClient, enabled):
+        self._enabled = enabled
         self.client = client
 
+    @property
+    def enabled(self) -> bool:
+        """Check if the service has everything it needs to work."""
+
+        return bool(self._enabled and self.client)
```

Comment:
This is going to get a bunch of things added to it pretty soon. This keeps all the changes here instead of distributed about.